R/utilities-simulation.R

```diff
@@ -173,7 +173,7 @@ simulateModelParallel <- function(structureSets,
 
     simulations <- lapply(structureSetsSubset, function(set) {
       re.tStoreFileMetadata(access = "read", filePath = set$simulationSet$simulationFile)
-      simulation <- loadSimulationWithUpdatedPaths(set$simulationSet,loadFromCache = TRUE)
+      simulation <- loadSimulationWithUpdatedPaths(set$simulationSet,loadFromCache = FALSE)
       logWorkflow(

```

Comment:
This is a temporary fix to the failing build until [this issue](https://github.com/Open-Systems-Pharmacology/OSPSuite-R/issues/733) is addressed.

Reply:
I don't understand. Are you running the same simulation twice?

Reply:
I can only reproduce if you are loading the SAME Simulation twice and running it in parallel. While it should not crash, I do not see any reason to do this as you are basically wasting CPU time.  
So the question is: Why do you need to run the same simulation using `runSimulations` multiple times?
Maybe @pchelle can answer this as well

Reply:
As mentioned [here](https://github.com/Open-Systems-Pharmacology/OSPSuite-R/issues/733), this threw an error only when I changed `loadFromCache` to `TRUE`.  When it was `FALSE` previously, this vignette example worked fine.  We can discuss whether simulations should be loaded from cache at this point.